src/loaders/LoadingManager.js

```diff
@@ -269,6 +289,36 @@ class LoadingManager {
 
 		};
 
+		/**
+		 * Can be used to abort ongoing loading requests in loaders using this manager.
+		 * The abort only works if {@link LoadingManager#enableAbortManagement} is set
+		 * to `true` and the loaders implement {@link Loader#abort}.
+		 *
+		 * @fires LoadingManager#abort
+		 * @return {LoadingManager} A reference to this loading manager.
+		 */
+		this.abort = function () {
+
+			this.dispatchEvent( _abort );
+
+			return this;
+
+		};
+
+		/**
+		 * Frees internal resources. Call this method whenever this instance is no
+		 * longer used in your app.
+		 *
+		 * @return {LoadingManager} A reference to this loading manager.
+		 */
+		this.dispose = function () {
+
+			this._listeners = {}; // remove abort handler
```

Comment:
A few days ago, I had this idea. Could we add the `dispose(type?: string)` method to the `EventDispatcher`